chalice/app.py

```diff
@@ -317,12 +349,30 @@ def __eq__(self, other):
         return self.__dict__ == other.__dict__
 
 
+class APIGateway(object):
+
+    _DEFAULT_BINARY_TYPES = [
```

Comment:
Yea I added a decent number more after this PR which will be in the next version.